api/volumes.go

```diff
@@ -129,11 +129,20 @@ func (a *api) handleGETVolumes(c jape.Context) {
 	if !a.checkServerError(c, "failed to get volumes", err) {
 		return
 	}
-	var jsonVolumes []VolumeMeta
-	for _, volume := range volumes {
-		jsonVolumes = append(jsonVolumes, toJSONVolume(volume))
+	var responseFormat string
+	if err := c.DecodeForm("response", &responseFormat); err != nil {
+		return
+	}
+	switch responseFormat {
```

Comment:
Stay consistent with the other endpoints and make the response type `VolumeResp []VolumeMeta` instead so there's no custom switch
logic in the endpoint.

Reply:
The additional content added by `toJSONVolume` is not used Prometheus, specifically the `vol.Errors` its iterating and adding to the response for the non Prometheus variant.

Reply:
It's fine if the volume errors aren't part of the Prometheus encoding. Just looking for consistency in the endpoints.

Reply:
resolved in next commit